src/xenia/gpu/dxbc_shader_translator_om.cc

```diff
@@ -12,11 +12,17 @@
 #include <cstdint>
 
 #include "xenia/base/assert.h"
+#include "xenia/base/cvar.h"
 #include "xenia/base/math.h"
 #include "xenia/gpu/draw_util.h"
 #include "xenia/gpu/render_target_cache.h"
 #include "xenia/gpu/texture_cache.h"
 
+DEFINE_double(fuzzy_alpha_epsilon,
```

Comment:
I strongly suggest to just define some constant epsilon in code and create boolean config value. 
This will prevent users from providing incorrect value and I expect every Nvidia GPU have exactly the same diff.

Reply:
Changed to bool in the new version and changed the name slightly, default epsilon is 0.01.